app/models/filters/hud_filter_base.rb

```diff
@@ -59,6 +59,28 @@ def effective_project_ids
       @effective_project_ids.uniq.reject(&:blank?)
     end
 
+    # Limit the effective project ids to only those with enrollments that overlap the report range
+    # OR
+    # Projects with operating dates overlapping the range.
+    # Looking for both of these will enable reporting on non-HMIS participating projects open during
+    # the report range, and will allow the data quality checks for enrollments that are open
+    # outside of the operating end dates
+    def effective_project_ids_during_range(effective_range)
+      @effective_project_ids_during_range ||= begin
```

Comment:
This seems like it could  lead to errors down the road - if this is called repeatedly with different `effective_range`s, it will always return the first one. Are we confident that  won't happen?
And semi related question, why can't we pull the `effective_range` from the start/end dates on the Filter?

Reply:
So, I don't think we'd ever call this with different ranges in the same iteration, but good point, I'd added the memoization before realizing we needed to add the range.  I'll adjust to memoize based on range.

We need the range because the issue it's fixing is related to the HIC, where HUD guidance of "run a 1 day LSA" isn't actually what they mean.  They mean, pull the data for a 1 day range and then run the LSA against it.  Our 1 day LSA still had a 7 year look-back. (Which is totally unnecessary for the HIC.)

Reply:
Gotcha. Sounds good. Thanks!